data-ingestion-apis/opentelemetry-metrics.mdx

```diff
@@ -93,139 +137,9 @@ curl -X POST "https://demo.middleware.io/v1/metrics" \
-
-To send custom logs to Middleware, POST to the following endpoint.
-
-```post POST
-POST https://<UID>.middleware.io:443/v1/logs
-```
-
-View the following example of a curl request sending custom logs to Middleware.
-```shell Shell
-curl -X POST "https://demo.middleware.io:443/v1/logs" \
--H "Accept: application/json" \
--H "Content-type: application/json" \
--d @- << EOF
-{
-   "resource_logs": [
-      {
-         "resource": {
-            "attributes": [
-               {
-                  "key": "mw.account_key",
-                  "value": {
-                     "string_value": "xxxxxxxxxx"
-                  }
-               },
-               {
-                  "key": "mw.resource_type",
-                  "value": {
-                     "string_value": "custom"
-                  }
-               },
-               {
-                  "key": "service_name",
-                  "value": {
-                     "string_value": "nginx-123"
-                  }
-               }
-            ]
-         },
-         "scope_logs": [
-            {
-               "log_records": [
-                  {
-                     "severity_text": "WARN",
-                     "body": {
-                        "string_value": "upstream server not accepting request"
-                     },
-                     "severity_number": "11",
-                     "attributes": [
-                        {
-                           "key": "server",
-                           "value": {
-                              "string_value": "nginx"
-                           }
-                        }
-                     ],
-                     "time_unix_nano": "1694030143000"
-                  }
-               ]
-            }
-         ]
-      }
-   ]
-}
-EOF   
-```
-
-### Resource Attributes
-
-| **Field** | **Required** | **Description** |
-| :-------: | :----------: | :------------- |
-| mw.account_key | ✅ | Authenticates into your Middleware Account |
-| mw.resource_type | ✅ | Groups and labels the log source, see [Resource Types](#resource-types) for allowed values |
-| service_name | ✖️ | Additional parameter to identify and filter logs |
-
-### Log Records
-
-| **Field** | **Required** | **Description** |
-| :-------: | :----------: | :------------- |
-| severity_text |  ✖️ | Labels log severity (string), requires one of FATAL, ERROR, WARN, INFO, DEBUG, TRACE |
-| [severity_number](#severity-number) |  ✖️ | Labels log severity (integer), number must be between 1 and 24 |
-| body | ✅ | The actual log body to ingest |
-| time_unix_nano | ✅ | Sets a timestamp for the log |
-
-#### severity_number
-|||
-| :-------------: | :--------: |
-| 1-4 | TRACE |
-| 5-8 | DEBUG |
-| 9-12 | INFO |
-| 13-16 | WARN |
-| 17-20 | ERROR |
-| 21-24 | FATAL |
+### Step 3: Send Data to Middleware
+Once you have setup your resource attributes, initialize your providers, and configure your exporters, begin sending data to Middleware. 
```

Comment:
Let's add an example of how each of those tracers may be used. I know Ed has some somewhere that can be asily plopped in.

"Once you have setup....some example calls using {w/e language ed is using} is below"

Reply:
I will add this to the guides section